src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/RhizomeUtils.java

```diff
@@ -101,7 +151,8 @@ public static void writeByteArray( ObjectOutput out, byte[] bytes ) throws IOExc
 
         public static String loadResourceToString( final String path ) {
             String resource = null;
-            try ( final InputStream stream = Thread.currentThread().getContextClassLoader().getResourceAsStream( path ) ) {
+            try ( final InputStream stream = Thread.currentThread().getContextClassLoader()
```

Comment:
Guava Resources does most of this already. Should probably update calls to use guava instead.